src/components/header/header.js

```diff
@@ -21,15 +21,13 @@ const AnimatedContainer = posed.div({
   },
 });
 
-const mainSiteUrl = process.env.SITE_URL || 'https://ssw.com.au';
-
-const Header = ({ displayActions, profileId }) => {
+const Header = ({ displayActions, profileId, parentSiteUrl }) => {
```

Comment:
Hey @patriciabarrosssw I'd import the site config instead of having it passed in

Reply:
done